src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -2783,9 +2783,51 @@ public final class JavadocTokenTypes {
     public static final int PARAM_TAG = JavadocParser.RULE_paramTag + RULE_TYPES_OFFSET;
 
     /**
-     * HTML void element {@code <embed>}.
+     * HTML void element.
      *
-     * @see #SINGLETON_ELEMENT
+     * <p><b>Example:</b></p>
+     * <pre>{@code
+     * <embed src="URL" type="MIME_type" width="width_value" height="height_value">
+     * }</pre>
+     * <b>Tree:</b>
+     * <pre>
+     * {@code
+     *   `--JAVADOC -> JAVADOC
+     *        |--NEWLINE -> \r\n
+     *        |--LEADING_ASTERISK ->  *
+     *        |--TEXT ->
+     *        |--HTML_ELEMENT -> HTML_ELEMENT
```

Comment:
let's start subtree with `HTML_ELEMENT` element to reduce the example AST tree, you can delete parent nodes